### ISSUE TITLE
Bad XML attribute in root

### ISSUE THREAD

**Issue by ThePix** [klembot/twinejs#489]
It would be useful to be able to read a Twine file into an XML parser (say for testing stuff). However, you cannot do that "as is" because the `hidden` attribute has no value. This is the default root tag:
```
<tw-storydata name="Tester" startnode="1" creator="Twine" creator-version="2.2.1" ifid="25535A0D-3CD8-45FA-BF26-580F3494A89F" zoom="1" format="Harlowe" format-version="2.1.0" options="" hidden>
```
Just setting it to an empty string seems to be fine (should it have a specific value?). But then when I use Twine to edit, it reverts.
```
<tw-storydata name="Tester" startnode="1" creator="Twine" creator-version="2.2.1" ifid="25535A0D-3CD8-45FA-BF26-580F3494A89F" zoom="1" format="Harlowe" format-version="2.1.0" options="" hidden="">
```
Using Twine 2.2.1 off-line on Win10.

Bonus points if the file gets a .xml extension - it is not HTML!

**Comment by klembot** [comment]
I can say that this is unlikely to happen, as the contents of passages very easily could be invalid XML and trying to clean that up seems at first glance difficult. Is there a reason you can't use an HTML parser instead?

**Comment by ThePix** [comment]
It is not HTML, is it?

I thought all the passage content was converted to be XML safe (e.g., `<` is saved as `&lt;`) so it should be valid XML. I was doing this on a game I made with plenty of JavaScript and a scattering of HTML in the passages, and this one attribute was the only thing making it invalid XML.

That said, I altered my program to modify the data as it is read in, and then process the XML, so I am not that bothered for myself.
